_posts/2017-09-26-sre-status.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: MozMEAO SRE Status Report - September 26, 2017
+author: Dave Parfitt
+excerpt_separator: <!--more-->
+---
+
+Here's what happened on the MozMEAO SRE team from September 19th - September 26th.
+
+<!--more-->
+
+## Current work
+
+### MDN Migration to AWS
+
+We've successfully completed a series of tests against MDN hosted in AWS. 
+
+#### Testing
+
+- [Manually test maintenance mode in stage AWS](https://github.com/mozmeao/infra/issues/498) (success 🎉)
+- [Manually test maintenance mode in prod AWS](https://github.com/mozmeao/infra/issues/515) (success 🎉)
+- [Manually test maintenance mode in SCL3 stage](https://github.com/mozmeao/infra/issues/510) (success 🎉)
+- [Maintenance mode traffic test: 5% to K8s](https://github.com/mozmeao/infra/issues/516) (success 🎉)
+- [Maintenance mode traffic test: 15% /50% / 100%](https://github.com/mozmeao/infra/issues/518) (success 🎉)
```

Comment:
please add a space before 50% in ``15% / 50% / 100 %``

Reply:
fixed